apps/DesktopStreamer/AppNapSuspender.h

```diff
@@ -34,7 +34,7 @@
 /* The views and conclusions contained in the software and           */
 /* documentation are those of the authors and should not be          */
 /* interpreted as representing official policies, either expressed   */
-/* or implied, of The University of Texas at Austin.                 */
+/* or implied, of The Ecole Polytechnique Federal de Lausanne.       */
```

Comment:
I'm not sure if we should use the accents, but we definitely should put a final e: FédéralE